Dockerfile

```
@@ -0,0 +1,40 @@
+ENV PROJECT_NAME=style-my-closet \
+    PROJECT_VERSION=1.0.0 \
+    JVM_OPTS=""
+
+# 빌드 스테이지에서 jar 파일만 복사
+COPY --from=builder /app/build/libs/${PROJECT_NAME}-${PROJECT_VERSION}.jar ./
+
+# 80 포트 노출
+EXPOSE 80
+
+# jar 파일 실행
+ENTRYPOINT sh -c "java ${JVM_OPTS} -jar ${PROJECT_NAME}-${PROJECT_VERSION}.jar"
```

Hadolint (2.12.0)

[warning] Use arguments JSON notation for CMD and ENTRYPOINT arguments

(DL3025)